domains/encrypt0r.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "main domain",
+  "owner": {
+    "username": "encrypt0r",
+    "email": "encrypt0r-x@protonmail.com"
+  },
+  "record": {
+    "TXT": "just_for_reservation"
```

```suggestion
    "TXT": "just_for_reservation"
```
You must have a complete and accessible website. A TXT record does nothing, you cannot just reserve a domain.